README.md

```diff
@@ -1,8 +1,21 @@
 # Stedi EDI Bootstrap
 
-This repo contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one possible way to interact with Stedi's Products and APIs to achieve a typical EDI workload; your implementation may include some or all of these products depending on your particular systems and requirements.
+This repository contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one way to build an integration for the common read and write EDI use case. Your solution may differ, depending on your systems and requirements. We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you design a custom integration.
```

Comment:
nit: `s/common read and write EDI use case/common read and write EDI use cases/`

also, I'm not sure about this phrasing: "design a custom integration." maybe something like "configure these workflows to suit your needs" (or something along those lines). cc: @joost-basic-bits for suggestions as well 😄

Reply:
> nit: s/common read and write EDI use case/common read and write EDI use cases/

Depends on whether you see reading and writing EDI as a single use case or as two.

> I'm not sure about this phrasing: "design a custom integration."

We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you customize the workflow.

Although, what you wrote, works too, Rob. @Laura, now you have options. :smile:

Reply:
thank you both! :). Design a custom integration is phrasing I think I've seen elsewhere (maybe??) but I really like the suggested edits. I will update to reflect!

Reply:
also - good call on making use cases plural!